sksparse/cholmod.pyx

```diff
@@ -86,18 +95,29 @@ cdef extern from "cholmod.h":
         int packed
 
     int cholmod_free_sparse(cholmod_sparse **, cholmod_common *) except? 0
+    int cholmod_l_free_sparse(cholmod_sparse **, cholmod_common *) except? 0
+
     int cholmod_check_sparse(cholmod_sparse *, cholmod_common *) except? 0
-    int cholmod_print_sparse(cholmod_sparse *, char *, cholmod_common *) except? 0
+    int cholmod_l_check_sparse(cholmod_sparse *, cholmod_common *) except? 0
+
+    int cholmod_print_sparse(cholmod_sparse *, const char *, cholmod_common *) except? 0
+    int cholmod_l_print_sparse(cholmod_sparse *, const char *, cholmod_common *) except? 0
 
     ctypedef struct cholmod_dense:
         size_t nrow, ncol, nzmax
         size_t d
         void * x
-        int xtype, dtype
+        int xtype
```

Comment:
Leave them on the same line?  Actually also merge the size_t declarations above, and the int fields declared for the Common class.